src/DataFrame/Operations/Aggregation.hs

```diff
@@ -120,7 +120,7 @@ aggregate aggs gdf@(Grouped df groupingColumns valueIndices offsets) =
             selectIndices
                 (VG.map (valueIndices VG.!) (VG.init offsets))
                 (select groupingColumns df)
-        groupedColumns = columnNames df L.\\ groupingColumns
```

Comment:
This is fine to remove. Previous approach generated all aggregations and only selected the interesting ones. But newer approach has that information baked into the aggregation.